demos/src/demo-step.js

```diff
@@ -1,28 +1,26 @@
-'use strict';
+import { select, selectAll } from 'd3-selection';
+import { default as PubSub } from 'pubsub-js';
 
-const d3Selection = require('d3-selection');
-const PubSub = require('pubsub-js');
+import step from './../../src/charts/step';
+import miniTooltip from './../../src/charts/mini-tooltip';
+import { StepDataBuilder } from './../../test/fixtures/stepChartDataBuilder';
 
-const step = require('./../../src/charts/step');
-const miniTooltip = require('./../../src/charts/mini-tooltip');
-
-const dataBuilder = require('./../../test/fixtures/stepChartDataBuilder');
-let redrawCharts;
+require('./helpers/resizeHelper');
```

Comment:
interesting, is this a mokey patch ?

Reply:
It only triggers the 'resize' event on the PubSub module, so the rest of demos react to it.